### PR TITLE
default seed to random.Random().getrandbits(32)

### LINKED ISSUE #267: seed based on best avialable random source (os.urandom or time.time() + pid + time.monotonic())
cpython defaults to using (os.urandom or time.time() + pid + time.monotonic()) for a random.seed: https://github.com/python/cpython/blob/d0981e61a5869c48e0a70a512967558391272a93/Modules/_randommodule.c#L233-L262

these functions aren't actually available to python code, but `random.Random().getrandbits(64)` seems to be the next best thing.

### PR DESCRIPTION
**Pull request by graingert**
Fixed #267

### CODE REVIEW

**Comment by graingert** [comment]
when investigating a failure with pytest-randomly==1.2.3 and pytest-xdist==1.31 I was seeing some cases where every worker selected the same seed (because all processes started so quickly)

I don't know if this new default provides any other actual real world benefit, but it feels "more correct"

File: src/pytest_randomly.py [hunk 1]
```diff
@@ -46,7 +45,7 @@
     have_numpy = False
 
 
-default_seed = int(time.time())
+default_seed = random.Random().getrandbits(64)
```
Comment (graingert) [inline src/pytest_randomly.py:48]:
```suggestion
default_seed = random.Random().getrandbits(32)
```

File: src/pytest_randomly.py [hunk 2]
```diff
@@ -70,7 +69,7 @@ def pytest_addoption(parser):
         type=seed_type,
         help="""Set the seed that pytest-randomly uses (int), or pass the
                 special value 'last' to reuse the seed from the previous run.
-                Default behaviour: use int(time.time()), so the seed is
+                Default behaviour: use random.Random().getrandbits(64), so the seed is
```
Comment (graingert) [inline src/pytest_randomly.py:72]:
```suggestion
                Default behaviour: use random.Random().getrandbits(32), so the seed is
```

**Comment by adamchainz** [comment]
Why the change from 64 to 32?

**Comment by graingert** [comment]
It's too big for python 3.6

**Comment by graingert** [comment]
ah no it's not, just specific to numpy's random seed

**Comment by graingert** [comment]
```
 INTERNALERROR>   File "/home/runner/work/pytest-randomly/pytest-randomly/.tox/py36/lib/python3.6/site-packages/pytest_randomly.py", line 144, in _reseed
INTERNALERROR>     np_random.seed(seed)
INTERNALERROR>   File "mtrand.pyx", line 243, in numpy.random.mtrand.RandomState.seed
INTERNALERROR>   File "_mt19937.pyx", line 166, in numpy.random._mt19937.MT19937._legacy_seeding
INTERNALERROR>   File "_mt19937.pyx", line 180, in numpy.random._mt19937.MT19937._legacy_seeding
INTERNALERROR> ValueError: Seed must be between 0 and 2**32 - 1
```

**Comment by graingert** [comment]
@adamchainz https://github.com/pytest-dev/pytest-randomly/issues/269

**Pull request merged**